alvr/session/src/settings.rs

```diff
@@ -941,7 +941,7 @@ pub fn session_settings_default() -> SettingsDefault {
                 rate_control_mode: RateControlModeDefault {
                     variant: RateControlModeDefaultVariant::Cbr,
                 },
-                filler_data: false,
+                filler_data: true,
```

Comment:
Can you explain the reason for this?

Reply:
I explained in the commit message?

Reply:
Sorry I didn't see that. But your reason s wrong. Adaptive bitrate requires CBR but does not require filler data. We calculate the supported bitrate of the network as `packet size / send duration`. This will lead to 100% network usage if filler data is enabled, or less than that if filler data is disabled, with equivalent output quality.

Reply:
interesting, without it enabled at least on amd it causes some issues through it could be because of the other things